testgrid/config/config.yaml

```diff
@@ -1990,8 +1990,40 @@ dashboards:
   - name: periodic-kubeadm-gce-1.6
     test_group_name: periodic-kubernetes-e2e-kubeadm-gce-1-6
 
+# These are the release *blocking* tests.  These provide a valid binary signal
+# to gate releases (alpha, beta, official).
+# This list is used by kubernetes/release/lib/releaselib.sh to provide no/nogo
```

Comment:
Sorry I was being too cute. There is a typo that says the two signals are `no` or `nogo`.